### DISCUSSION TITLE
[HW4] Can we use `Trainer` to fine-tune the model?

### DISCUSSION THREAD

**Discussion by yuneg11** [seominjoon/kaist-ai605#158] [Q&A] [answered]
The link in hw4 specifies the section "fine-tuning-in-native-pytorch" that uses own train procedure based on python `for` iterations.
However, can we use `Trainer` class used above the tutorial?

**Comment by seominjoon** [comment, marked as answer]
Yes, you are allowed to use anything provided by Hugging Face.
